mvm/eos/engine.go

```diff
@@ -84,19 +83,26 @@ func Boot(conf *Configuration, threshold int) (*Engine, error) {
 	if err != nil {
 		panic(fmt.Errorf("Invalid private key: %s", conf.PrivateKey))
 	}
+	pubKey := key.GetPublicKey().String()
+	pubKeyVerified := false
 
 	if len(conf.PublicKeys) == 0 {
 		panic("public-keys not specified!")
 	}
 	pubs := make([]*secp256k1.PublicKey, 0, len(conf.PublicKeys))
 	for _, pub := range conf.PublicKeys {
+		pubKeyVerified = pubKeyVerified || (pub == pubKey)
```

Comment:
We need to compare public keys in raw format, 
```
		_pub, err := secp256k1.NewPublicKeyFromBase58(pub)
		if err != nil {
			panic(fmt.Errorf("Invalid public key: %s", pub))
		}
		pubKeyVerified = pubKeyVerified || (*_pub == *pubKey)
```
also, change `pubKey := key.GetPublicKey().String()` to 
```
pubKey := key.GetPublicKey()
```

There are two kinds of public keys in string formation like below:
```
PUB_K1_5vKpMYTByN6Mv28NZDQVnu29PPwrLGFCTcUs8oq6WsMmxLYtpm
EOS5vKpMYTByN6Mv28NZDQVnu29PPwrLGFCTcUs8oq6WsMmysLVtZ
```
so comparing public keys in string formation is inconsistent.